src/SFA.DAS.Employer.PR.Web/Views/DeclineCreateAccount/Index.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@using SFA.DAS.Employer.PR.Web.Models.Requests
+@using SFA.DAS.Employer.PR.Domain.Models
+
+@model DeclineCreateAccountViewModel
+﻿@{
+    ViewBag.Title = "Decline Create account";
+}
+@section breadcrumb {
+    <a class="das-js-back-link">Back</a>
+}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-form-group">
+            <h1 class="govuk-heading-l">
+                Are you sure you do not want to create an account?
+            </h1>
+            <p class="govuk-body">
+                @Model.ProviderName should have confirmed these details with you before sending the request to create an account. You can change your name by going back and selecting Change your name.
+            </p>
+            <p class="govuk-body">
+                If you select <strong>Do not create account</strong>, your apprenticeship service account will not be set up and we’ll remove the details added by @Model.ProviderName.
+                If the details were not correct, they can correct them and send you a new request.
+            </p>
+            <p class="govuk-body">
+            <form method="GET">
+                <input asp-for="RequestId" id="requestId" type="hidden" />
```

Comment:
Not sure why you need RequestId to post back? It should be there in the url?

Reply:
removed now